test/infrastructure/docker/docker/kind_manager.go

```diff
@@ -124,6 +124,8 @@ func createNode(name, image, clusterLabel, role string, mounts []v1alpha4.Mount,
 		"--label", clusterLabel,
 		// label the node with the role ID
 		"--label", fmt.Sprintf("%s=%s", nodeRoleLabelKey, role),
+		"--sysctl=net.ipv6.conf.all.disable_ipv6=0",
+		"--sysctl=net.ipv6.conf.all.forwarding=1",
```

Comment:
We should check kind default here or make this dynamic according to the cluster ipFamily

Reply:
related code in kind: https://github.com/kubernetes-sigs/kind/blob/b6bc112522651d98c81823df56b7afa511459a3b/pkg/cluster/internal/providers/docker/provision.go#L192-L195

Reply:
Thanks for the pointer! This clarifies this bit to me